content/community.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Ozone Community
+---
+
+## Ozone Mailing Lists
+
+### Developers
+
+If you’d like to contribute to Ozone, please subscribe to the Ozone developer mailing list.
+
+The Ozone developer mailing list is: dev@ozone.apache.org.
+
+* [Subscribe to List](mailto:dev-subscribe@ozone.apache.org)
+* [Unsubscribe from List](mailto:dev-unsubscribe@ozone.apache.org)
+* [Archives](http://mail-archives.apache.org/mod_mbox/ozone-dev/)
+
+### Commits
+
+If you’d like to see changes made in the Ozone version control system, please subscribe to the Ozone commits mailing list.
+
+The Ozone commits mailing list is: commits@ozone.apache.org.
+
+* [Subscribe to List](mailto:commits-subscribe@ozone.apache.org)
+* [Unsubscribe from List](mailto:commits-unsubscribe@ozone.apache.org)
+* [Archives](http://mail-archives.apache.org/mod_mbox/ozone-commits/)
+
+### Issues
+
+If you’d like to see changes made in the Ozone issue tracking system, please subscribe to the Ozone issues mailing list.
+
+The Ozone issues mailing list is: issues@ozone.apache.org.
+
+* [Subscribe to List](mailto:issues-subscribe@ozone.apache.org)
+* [Unsubscribe from List](mailto:issues-unsubscribe@ozone.apache.org)
+* [Archives](http://mail-archives.apache.org/mod_mbox/ozone-issues/)
+
+## Ozone Slack
+
+If you would like to join the Ozone slack Channel, Please use this link to get self-invited  https://s.apache.org/slack-invite and join the **#ozone** channel.
```

Comment:
I think "p" should be lowercase here.

Reply:
@flirmnave if you are making further changes, please also update http://mail-archives.apache.org links to HTTPS.  (I didn't want to ask for this change in itself...)